SWxTREC/msis
# SWT-209 add download options for altitude plot as well

This PR adds the same download capability that was available for the surface plot to the altitude plot. 
It also updates the deploy instructions to deploy twice (for now). 

Then I went and updated the packages. 

To test: `npm i && npm start`
You should be able to download the altitude plot data just like you can download the surface plot data
